cs2pr

```diff
@@ -34,6 +35,9 @@ foreach ($argv as $arg) {
         case 'colorize':
             $colorize = true;
             break;
+        case 'no-notices':
```

Comment:
Thx for the PR. I like the change in general.

I feel this option should be named `notices-as-warnings` or similar.
Additionally the new option should be added to the help output.

lastly this PR changes the mode-bits of this file - did this change intentionally?